_posts/2017-11-08-cost-of-drupal-7.md

```diff
@@ -0,0 +1,194 @@
+
+#### The Improved Developer Experience (DX) of Drupal 8
+
+Just like happy customers tend to be repeat customers, [happier developers](http://fortune.com/2015/10/29/happy-productivity-work/) [also produce returns](https://www.fastcompany.com/3048751/happy-employees-are-12-more-productive-at-work); they're [more productive](http://www.snacknation.com/blog/employee-happiness/).
+
+There are a few improvements in Drupal 8 that make life _substantially_ better for developers. The [Configuration Management Initiative](https://www.drupal.org/docs/8/configuration-management/managing-your-sites-configuration) was a boon to developers who struggled with a module called "features" which was not designed to do what most of us used it for. The CMI addresses the previous workaround, rife with inconsistencies, of moving site configuration from development to staging and production environments. Although it may seem trivial, developers _love this better system_ in Drupal 8, and it means more efficient development, therefore higher ROI.
+
+<!-- @todo: Get team feedback on this topic. Include composer too? It seems with both there have been some difficulties. Anything else? -->
+
+#### Proudly found elsewhere / not invented here / getting off the island
+
+A primary Drupal 8 philosophy that has largely been successful, but yet to fully bear fruit, is the concept to drastically reduce "Drupalisms" that had proven a challenge for newcomers to the system who had to learn a suite of things specific to only Drupal. The proudly found elsewhere paradigm seeks to mitigate this by leveraging the best of other open source tools when possible rather than reinvent the wheel. A few of the tools Drupal 8 now exploits are [Symfony’s HTTP Foundation classes](https://symfony.com/), [Twig templating engine](https://twig.symfony.com/), and [Composer Dependency Manager](https://getcomposer.org/). This "borrowing" has two positive consequences: 1) it reduces the workload for Drupal core contributors by utilizing what's freely available and well vetted through other communities, 2) it allows people familiar with those other frameworks a smoother onramp to productivity in Drupal. I believe we haven't yet seen a majority of the benefit to the Drupal 8 project from the many people who were already versed in Symfony and TWIG before working with Drupal.
```

Comment:
Drupal 8 actually uses a bunch of Symfony components, not just HTTP foundation. Here's a full list: https://symfony.com/projects/drupal. Perhaps change `[Symfony’s HTTP Foundation classes](https://symfony.com/)` to `[Symfony components like HTTP Foundation]https://symfony.com/projects/drupal)`